doc/Preboarding_and_Onboarding.rst

```diff
@@ -25,6 +25,7 @@ The preboarding procedure must be performed only once. On the preboarding proces
 
 1. Open a browser and navigate to the *https://<management IP>/#/preboarding/welcome*
 
+   - If you use OVA 1.0.4 or earlier, you should use *https://<management IP>:28100/#/preboarding/welcome*
```

Comment:
I think this line was already added in a different PR.  Can you check please if still needed?

Reply:
Hi @johnbent, this is same PR, with new addition, because @mukul-seagate11 has a change request to update the guide for for 1.0.4 release.

Reply:
@wb-droid, this change has been fix in https://github.com/Seagate/cortx/pull/1134